src/makeInfinityAdder.js

```diff
@@ -4,7 +4,17 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  const adder = (sum = 0) => {
+    return (value) => {
+      if (value === undefined) {
```

Comment:
The check `if (value === undefined)` is used to determine if the function should return the current sum. This works for cases where no argument is passed, but if `undefined` is explicitly passed as an argument, it will also trigger this condition. Consider using a different approach to distinguish between no argument and an explicit `undefined` value, such as checking the number of arguments passed.